examples/wasm-demo/www/package-lock.json

```diff
@@ -0,0 +1,5909 @@
+{
```

Comment:
Not really familiar with Web, but if this is generated, I assume this shouldn't be a part of the repository.

Reply:
It's used for reproducible builds: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5
Though some think it can be omitted.

Reply:
Gotcha, seems `Cargo.lock` stuff. This seems to be able to remove safely in most of the case.